test/core/test_sequencer.py

```diff
@@ -1,19 +1,35 @@
-import time
-import unittest
-from unittest.mock import MagicMock, call
-from pubsub import pub
+import pytest
 import fixate
-from fixate.core.common import TestList as FixateTL, TestClass as FixateTC
+from fixate.core.common import TestList, TestClass
+from fixate.core.checks import chk_fails, chk_passes
+from fixate.core.exceptions import SequenceAbort
```

Comment:
You've imported `SequenceAbort` but you haven't used it.

Reply:
Removed.